modules/outbound-command-event-handler/test/integration/application/process_sdk_outbound_bulk_quotes_request.test.ts

```diff
@@ -0,0 +1,396 @@
+    expect(hasAcceptPartyEvent).toBeTruthy();
+
+    // Command event for sdk outbound accept bulk party info request
+    const processSDKOutboundBulkAcceptPartyInfoCommandEventData : IProcessSDKOutboundBulkAcceptPartyInfoCmdEvtData = {
+      bulkId: bulkTransactionId,
+      bulkTransactionContinuationAcceptParty: {
+        bulkHomeTransactionID: 'string',
+        individualTransfers: [
+          {
+            homeTransactionId: 'string',
+            transactionId: randomGeneratedTransferIds[0],
+            acceptParty: true
+          },
+          {
+            homeTransactionId: 'string',
+            transactionId: randomGeneratedTransferIds[1],
+            acceptParty: false
+          },
+          {
+            homeTransactionId: 'string',
+            transactionId: randomGeneratedTransferIds[2],
+            acceptParty: true
+          }
+        ]
+      },
+      timestamp: Date.now(),
+      headers: []
+    }
+    const processSDKOutboundBulkAcceptPartyInfoCommandEventObj = new ProcessSDKOutboundBulkAcceptPartyInfoCmdEvt(
+      processSDKOutboundBulkAcceptPartyInfoCommandEventData
+    );
+    await producer.sendCommandEvent(processSDKOutboundBulkAcceptPartyInfoCommandEventObj);
+    await new Promise(resolve => setTimeout(resolve, messageTimeout));
+
+    // Check that the global state to be DISCOVERY_ACCEPTANCE_COMPLETED
+    const bulkStateTwo = await bulkTransactionEntityRepo.load(bulkTransactionId);
+    expect(bulkStateTwo.state).toBe(BulkTransactionInternalState.DISCOVERY_ACCEPTANCE_COMPLETED);
+
+    // Check that individual transfer to receiverfsp with acceptPart true have been updated to DISCOVERY_ACCEPTED
+    const afterIndividualTransfer1 = await bulkTransactionEntityRepo.getIndividualTransfer(bulkTransactionId,  randomGeneratedTransferIds[0]);
+    expect(afterIndividualTransfer1.state).toBe(IndividualTransferInternalState.DISCOVERY_ACCEPTED);
+
+    // Check that individual transfer to receiverfsp with acceptParty false have been updated to DISCOVERY_REJECTED
+    const afterIndividualTransfer2 = await bulkTransactionEntityRepo.getIndividualTransfer(bulkTransactionId,  randomGeneratedTransferIds[1]);
+    expect(afterIndividualTransfer2.state).toBe(IndividualTransferInternalState.DISCOVERY_REJECTED);
+
+    // Check that individual transfer to differentfsp with acceptParty true have been updated to DISCOVERY_ACCEPTED
+    const afterIndividualTransfer3 = await bulkTransactionEntityRepo.getIndividualTransfer(bulkTransactionId,  randomGeneratedTransferIds[1]);
+    expect(afterIndividualTransfer3.state).toBe(IndividualTransferInternalState.DISCOVERY_REJECTED);
+
+    // Check that command handler sends event to domain handler
+    const hasSDKOutboundBulkAcceptPartyInfoProcessed = (domainEvents.find((e) => e.getName() === 'SDKOutboundBulkAcceptPartyInfoProcessedDmEvt'));
+    expect(hasSDKOutboundBulkAcceptPartyInfoProcessed).toBeTruthy();
```

Comment:
Instead of checking for Truthy, can we assert that only one event is published for the DISCOVERY_ACCEPTED state. Want to make sure that events are not being published for DISCOVERY_REJECTED state.